README.md

```diff
@@ -7,25 +7,11 @@ ocitools is a collection of tools for working with the [OCI runtime specificatio
 [`ocitools generate`][generate.1] generates [configuration JSON][config.json] for an [OCI bundle][bundle].
 [OCI-compatible runtimes][runtime-spec] like [runC][] expect to read the configuration from `config.json`.
 
-```sh
-$ ocitools generate --output config.json
-$ cat config.json
-{
-        "ociVersion": "0.5.0",
-        …
-}
-```
-
 ## Validating an OCI bundle
 
 [`ocitools validate`][validate.1] validates an OCI bundle.
 The error message will be printed if the OCI bundle failed the validation procedure.
 
-```sh
-$ ocitools generate
-$ ocitools validate
-INFO[0000] Bundle validation succeeded.
-```
 
 ## Testing OCI runtimes
```

Comment:
On Fri, Jul 22, 2016 at 09:43:41AM -0700, Antonio Murdaca wrote:
> >  ## Testing OCI runtimes
> 
> I guess the question is: shouldn't testruntime.sh be a part of ocitools? 

Yes, but it's hard to do right ;).  See discussion in #61 and #98 (I'm
partial to [1] ;).

[1]: https://github.com/opencontainers/ocitools/pull/61#issuecomment-217910342